### PR TITLE
EWPP-354: Fixing display exports.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by nagyad — APPROVED** [review]

**Review by nagyad — DISMISSED** [review]

File: translations/oe_content-ro.po [hunk 1]
```diff
@@ -201,3 +201,123 @@ msgstr "Vineri"
 
 msgid "Saturday"
 msgstr "Sâmbătă"
+
+msgid "Email"
```
Comment (upchuk) [inline translations/oe_content-ro.po:205]: None of these are translated!! Please verify each language in each PO file and ensure all are translated.

**Review by upchuk — CHANGES_REQUESTED** [review]

File: translations/oe_content-ro.po (hunk 1 above)
Comment (upchuk) [reply translations/oe_content-ro.po:205]: It seems 3 languages are missing translations

**Review by upchuk — CHANGES_REQUESTED** [review]
Why isn't `Practical information` translated for example? It is in Event sheet of the combined file

File: translations/oe_content-ro.po (hunk 1 above)
Comment (sergepavle) [reply translations/oe_content-ro.po:205]: Should be fixed. At least correlated with combined.xlsx file (original file with updated translations from Tamas)

**Comment by sergepavle** [comment]
> Why isn't `Practical information` translated for example? It is in Event sheet of the combined file

Practical information string has translation in the oe_theme as string translation.

File: translations/oe_content-cs.po [hunk 2]
```diff
@@ -201,3 +201,102 @@ msgstr "Pátek"
...
+msgid "Closed"
+msgstr "ukončeno"
+
+msgid "Details"
+msgstr "Podrobnosti"
+
+msgid "Status"
+msgstr "Status"
+
+msgid "Details"
```
Comment (upchuk) [inline translations/oe_content-cs.po:280]: Details is duplicated. Check everywhere.

**Review by upchuk — CHANGES_REQUESTED** [review]
The string `Organisation` is missing. You can find it in the `Contact` sheet of the XLS file...

How can some of the strings be missing from the PO file but not all of them?

Also `Mobile number`, `Office`.

Can you please ensure all the strings in the `Contact` sheet are translated?

**Comment by sergepavle** [comment]
> Also `Mobile number`, `Office`.

These strings have been removed from final xls and po files because I didn't find usage of these strings in the frontend. I put them back in the HandledOriginalCombined.xlsx file (blue background).

File: translations/oe_content-cs.po (hunk 2 above)
Comment (sergepavle) [reply translations/oe_content-cs.po]: Fixed

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]
the oe_content_news.info file needs to be updated to reflect we no longer ship with the teaser view mode.

**Review by ademarco — DISMISSED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by ademarco — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**